### DISCUSSION TITLE
[BUG] Issue title...Results do not correspond to current coco set

### DISCUSSION THREAD

**Discussion by Aliweka2020** [rwightman/efficientdet-pytorch#264] [Q&A]
I am creating a jupyter notebook to train and validate DETR using efficinet architecture on coco dataset.
while evaluating this error apear

~\anaconda3\lib\site-packages\pycocotools\coco.py in loadRes(self, resFile)
324 assert type(anns) == list, 'results in not an array of objects'
325 annsImgIds = [ann['image_id'] for ann in anns]
--> 326 assert set(annsImgIds) == (set(annsImgIds) & set(self.getImgIds())),
327 'Results do not correspond to current coco set'
328 if 'caption' in anns[0]:

AssertionError: Results do not correspond to current coco set

any help ..?

**Comment by rwightman** [comment]
@Aliweka2020 sorry, I won't be able to help with that, looks like maybe a bad (old?) version of pycocotools or somethign wrong with the dataset. I'll leave open in case anyone else can help but will close soon as it's unlikely to be a bug here.
